scripts/lib/mlkem_test.py

```diff
@@ -526,29 +525,32 @@ def init_results() -> TypedDict:
 
         return fail
 
+    def _acvp(self, opt: bool, compmile: bool, run: bool, acvp_dir: str) -> bool:
```

Comment:
Why is this handled differently from say `_func`? In `_func`, you just call `self._f(...).test()`, which bundles `compile` and `run` -- while here you still call both separately. Note also that L530 and L532 do not operate on the same test object -- they call `_f` twice.

Reply:
because the way we run acvp is very different from the other tests, the other tests only run once for each parameter set, but acvp test is run multiple times for each parameter set with different test vectors.
it's too hard and not necessary to reuse the same function `test` for now.